script.js

```diff
@@ -1,4 +1,145 @@
+    (userInput == "stone" && gameResult == "scissors")
+  ) {
+    return `You chose ${newFormat}. <br> The AI chose ${rspFormat}! <br> You win. Play again?`;
+  }
+
+  // lose condition
+  if (
+    (userInput == "scissors" && gameResult == "stone") ||
+    (userInput == "paper" && gameResult == "scissors") ||
+    (userInput == "stone" && gameResult == "paper")
+  ) {
+    return `You chose ${newFormat}. <br> The AI chose ${rspFormat}!<br> You lose. Play again?`;
+  }
+  //**REVERSE GAME */
+  //draw condition
+  if (
+    (userInput == "reverse scissors" && gameResult == "scissors") ||
+    (userInput == "reverse paper" && gameResult == "paper") ||
+    (userInput == "reverse stone" && gameResult == "stone")
+  ) {
+    return `You chose ${newFormat}. <br> The AI chose ${rspFormat}! <br> It's a draw. <br> Play again?`;
+  }
+  // lose condition
+  if (
+    (userInput == "reverse scissors" && gameResult == "paper") ||
+    (userInput == "reverse paper" && gameResult == "stone") ||
+    (userInput == "reverse stone" && gameResult == "scissors")
+  ) {
+    return `You chose ${newFormat}. <br> The AI chose ${rspFormat}! <br> You lose. Play again?`;
+  }
+
+  // win condition
+  if (
+    (userInput == "reverse scissors" && gameResult == "stone") ||
+    (userInput == "reverse paper" && gameResult == "scissors") ||
+    (userInput == "reverse stone" && gameResult == "paper")
+  ) {
+    return `You chose ${newFormat}. <br> The AI chose ${rspFormat}!<br> You lose. Play again?`;
+  }
+  //**REVERSE GAME */
+
+  // if wrong input, prompt user to try again
+  else if (
+    userInput != "scissors" ||
+    userInput != "paper" ||
+    userInput != "stone" ||
+    userInput != "reverse scissors" ||
+    userInput != "reverse paper" ||
+    userInput != "reverse stone"
+  ) {
+    return `Oops! You have input the wrong option. Please only type in "scissors" or "paper" or "stone" to join the game. Try again!`;
+  }
+  return rspResult;
+};
+
+// program generates rsp
+
+var aiResult = function () {
+  var rspNumber = 1 + Math.floor(Math.random() * 3);
+  console.log(rspNumber);
+
+  if (rspNumber == 1) {
+    rspResult = "scissors";
+    rspFormat = "scissors ✂️";
+  }
+
+  if (rspNumber == 2) {
+    rspResult = "paper";
+    rspFormat = "paper 🗒";
+  }
+
+  if (rspNumber == 3) {
+    rspResult = "stone";
+    rspFormat = "stone O";
+  }
+  return rspResult;
+};
+
+// program converts input into a different format
+var format = function () {
+  var newFormat = userInput;
+
+  if (userInput == "scissors") {
+    newFormat = "scissors ✂️";
+  }
+  if (userInput == "paper") {
+    newFormat = "paper 🗒";
+  } else if (userInput == "stone") {
+    newFormat = "stone O";
+  }
+
+  //convert reverse input
+  if (userInput == "reverse scissors") {
+    newFormat = "reverse scissors ✂️";
+  }
+  if (userInput == "reverse paper") {
+    newFormat = "reverse paper 🗒";
+  }
+  if (userInput == "reverse stone") {
+    newFormat = "reverse stone O";
+  }
```

Comment:
Good job! I really like how you managed to find a way to format your desired message all in one function! Excellent decomposition!

Reply:
Some were having trouble finding a way to put everything together in such a concise way!